### ISSUE TITLE
CWorldSolver

### ISSUE THREAD

**Issue by rcnlee** [JuliaPOMDP/ContinuumWorld.jl#5]
Should the integrated CWorldSolver be deprecated in favor of [LocalFunctionApproximation.jl](https://github.com/sisl/LocalFunctionApproximation.jl)? Or keep for pedogogical purposes?

**Comment by zsunberg** [comment]
Yes, it would be better to use LocalFunctionApproximation! I think no-one has had time to integrate it yet.

**Comment by dylan-asmar** [comment]
I was working through updating ContinuumWorld.jl and was thinking it should probably be a standalone model or integrated into POMDPModels. As it stands in its current state, it doesn't fit into the POMDPs.jl ecosystem. We could include the visualization of the value iteration in the docs or README as an example using LocalFunctionApproximation.jl.
